docs/src/pages/[platform]/components/rating/examples/RatingThemeExample.tsx

```diff
@@ -0,0 +1,26 @@
+import { Rating, Flex, ThemeProvider, Theme } from '@aws-amplify/ui-react';
+
+const theme: Theme = {
+  name: 'rating-theme',
+  tokens: {
+    components: {
+      rating: {
+        filled: { color: { value: '{colors.yellow.60}' } },
```

Comment:
Could use a more contrasty yellow. It doesn't meet 3:1 in light mode
![Screen Shot 2022-06-16 at 3 28 24 PM](https://user-images.githubusercontent.com/40295569/174189021-d8f8f971-cc91-4951-81bd-d58fcf4e8a99.png)